Source/WebCore/page/ChromeClient.h

```diff
@@ -59,6 +59,7 @@
 #include "WebGPU.h"
 #include "WorkerClient.h"
 #include <JavaScriptCore/ConsoleTypes.h>
+#include <cstdint>
```

Comment:
This include probably isn't needed. If clangd is adding these you can add `--header-insertion=never` to your language server arguments.